micronaut/src/main/java/com/gluonhq/ignite/micronaut/OnFxThreadInterceptor.java

```diff
@@ -0,0 +1,19 @@
+package com.gluonhq.ignite.micronaut;
+
+import io.micronaut.aop.MethodInterceptor;
+import io.micronaut.aop.MethodInvocationContext;
+import javafx.application.Platform;
+
+import javax.inject.Singleton;
+
+@Singleton
+class OnFxThreadInterceptor implements MethodInterceptor<Object, Object> {
```

Comment:
I'd recommend renaming to `onFXThreadInterceptor` as the other class is called `OnFXThread` (cap X)

Reply:
Updated